docs/src/main/paradox/index.md

```diff
@@ -19,7 +23,15 @@ Based on a protobuf service definition, akka-grpc can generate:
 * On the server side, code to create an Akka HTTP route based on your implementation of the API
 * On the client side, a client for the API.
 
+### gRPC vs REST
```

Comment:
Some discussion that it is "synchronous" in the sense that other side needs to be online when we make the call, but non-blocking server/client implementations lead to resource efficient hardware utilisation?

Reply:
That would be true for both gRPC and REST, right?

Reply:
Yes, as seen in the ongoing debate though this seems to be an area of confusion so perhaps worth calling out all these are synchronous, more or less decoupled, yet nonblocking implementations?

Reply:
Maybe that goes in a "when not to use it" section? Where we can hint towards message queues etc?